packages/legend-query-builder/src/components/data-access/DataAccessOverview.tsx

```diff
@@ -308,12 +327,13 @@ export const DataAccessOverview = observer(
           'data-access-overview--compact': Boolean(compact),
         })}
       >
-        <PanelLoadingIndicator
-          isLoading={
-            dataAccessState.surveyDatasetsState.isInProgress ||
-            dataAccessState.checkEntitlementsState.isInProgress
-          }
-        />
+        {isLoading && (
+          <>
+            <PanelLoadingIndicator isLoading={true} />
+            <PanelDivider />
```

Comment:
I don't think this is right

Reply:
As discussed leaving to add spacing so you get
<img width="1110" alt="image" src="https://github.com/finos/legend-studio/assets/41248956/596ab283-a399-4683-8113-4ad825cedb6d">
rather than
<img width="1093" alt="image" src="https://github.com/finos/legend-studio/assets/41248956/10b363bb-5822-446c-83d5-00bb12a7e152">